frontend/views/containers/access/LoginForm.vue

```diff
@@ -28,7 +28,8 @@ form(
   .buttons.is-centered
     i18n(
       tag='button'
-      :disabled='$v.form.$invalid'
+      :data-loading='ephemeral.isSubmitting'
+      :disabled='$v.form.$invalid || ephemeral.isSubmitting === "true"'
```

Comment:
@pieer and @taoeffect there's another way to keep `data-loading` and using Boolean. Transform the Bool into a String the moment it's added to the DOM. Everywhere else it can be a bool as usual.

```
:data-loading='(ephemeral.isSubmitting).toString()'
:disabled='$v.form.$invalid || ephemeral.isSubmitting'
```

What approach do you prefer?

Reply:
Is this question still relevant given what you wrote above re

> Done! `.is-loader` + `data-loading="Bool"`

?

Reply:
No, it's not. The question was written before finding the new solution ;)